### ISSUE TITLE
kafka-go cannot install

### ISSUE THREAD

**Issue by lasco2a** [segmentio/kafka-go#593]
Hi there,

Just trying to install kafka-go but getting below error:

go get github.com/segmentio/kafka-go
# github.com/segmentio/kafka-go/protocol
../../go/src/github.com/segmentio/kafka-go/protocol/record_batch.go:55:7: undefined: errors.Is
../../go/src/github.com/segmentio/kafka-go/protocol/record_batch.go:104:7: undefined: errors.Is
../../go/src/github.com/segmentio/kafka-go/protocol/record_batch.go:361:7: undefined: errors.Is
../../go/src/github.com/segmentio/kafka-go/protocol/record_v1.go:107:9: undefined: errors.Is

Cheers,
Laurent

**Comment by achille-roussel** [comment]
Hello @lasco2a, thanks for reporting the issue.

[errors.Is](https://golang.org/pkg/errors/#Is) was introduced in Go 1.13, are you attempting to build kafka-go with Go 1.12 or older?

**Comment by lasco2a** [comment]
Hi Achille,
I am using v1.12.10:
go-test$ go version
go version go1.12.10 linux/amd64

**Comment by achille-roussel** [comment]
Currently the kafka-go code requires Go 1.13 or later, as described by the `go.mod` file:

```
$ cat go.mod
module github.com/segmentio/kafka-go

go 1.13

require (
	github.com/eapache/go-xerial-snappy v0.0.0-20180814174437-776d5712da21
	github.com/golang/snappy v0.0.1
	github.com/klauspost/compress v1.9.8
	github.com/pierrec/lz4 v2.0.5+incompatible
	github.com/stretchr/testify v1.6.1
	github.com/xdg/scram v0.0.0-20180814205039-7eeb5667e42c
	github.com/xdg/stringprep v1.0.0 // indirect
	golang.org/x/crypto v0.0.0-20190506204251-e1dfcc566284 // indirect
	golang.org/x/net v0.0.0-20190404232315-eb5bcb51f2a3
)
```

Would upgrading to Go 1.13+ be a possibility?

**Comment by lasco2a** [comment]
Thanks a ton for your help.
I upgraded to v1.13.8 and it installed properly now.

**Comment by achille-roussel** [comment]
I'm glad you got it sorted out 👍

**Comment by prekratko** [comment]
Just as an info.
I also stumbled upon this problem. My version was 1.12.7 and I upgraded to the latest available, 1.15.8.
(because of dependencies, upgrade to 1.13 was not trivial so I chose the ultimate solution :) )
Compiling succeeded.
